README.md

```diff
@@ -134,6 +134,7 @@ The default access key is accessKey1 with
 a secret key of verySecretKey1.
 
 ## Run it with Vault user management
+Note: Vault is a private product and you will need access to it separately.
```

Comment:
```suggestion
Note: Vault is proprietary and must be accessed separately.
```

Reply:
I spent like 5 minutes thinking about how to best describe this, before I realized I had no idea what terms to use and just went with whatever... thanks

Reply:
No worries, matey. Words are what I do!